### ISSUE TITLE
Tools::getIsset(Tools::getValue('id_shop')) return false

### ISSUE THREAD

**Issue by BoutiquesInternet** [PrestaShop/gsitemap#101]
Hi,

In multistore mode, Tools::getIsset(Tools::getValue('id_shop')) return false;
line 52 in gsitemap-cron.php

gsitemap version : 4.1.0
Prestashop version : 1.7.4.3

**Comment by MathiasReker** [comment]
Can you make a PR to solve your issue?

**Comment by Quetzacoalt91** [comment]
Indeed, it seems we do not properly check a data is sent in the request header. `Tools::getIsset()` was enough to test.

**Comment by 01park** [comment]
Hi,

I was wondering if this issue was solved. 
I think I might have the same problem with the multistore. When a cron task is execute it always updates shop 1, eventhough I have multiple cron jobs for shop 2 - 3 - 4 etc
